web/web.go

```diff
@@ -263,7 +265,20 @@ func (h *Handler) statush(w http.ResponseWriter, r *http.Request) {
 	h.status.mu.RLock()
 	defer h.status.mu.RUnlock()
 
-	h.executeTemplate(w, "status", h.status)
+	h.executeTemplate(w, "status", struct {
+		Status *PrometheusStatus
+		Info   map[string]string
+	}{
+		Status: h.status,
+		Info:   version.Map,
+	})
+}
+
+func (h *Handler) version(w http.ResponseWriter, r *http.Request) {
```

Comment:
Guess you don't want this under `/api/v1` because it should be the same path for all API versions? Hmm...